core-feature-pack/src/main/resources/content/bin/common.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh -x
 
+setJava9_plus() {
+  $JAVA --add-modules=java.se -version > /dev/null 2>&1 && JAVA_9_PLUS=true || JAVA_9_PLUS=false
+}
+
 setDefaultModularJvmOptions() {
-  $JAVA --add-modules=java.se -version > /dev/null 2>&1 && MODULAR_JDK=true || MODULAR_JDK=false
-  if [ "$MODULAR_JDK" = "true" ]; then
+  if [ "$JAVA_9_PLUS" = "true" ]; then
```

Comment:
The `setJava9_plus` function, or whatever it ends up being called, should be invoked in this function to ensure the variable is set.